packages/truffle-db/src/workspace/test/contractInstances.graphql.ts

```diff
@@ -0,0 +1,94 @@
+import gql from "graphql-tag";
```

Comment:
I think this file should be renamed to the singular -- `contractInstance.graphql.ts`

Reply:
Oh yea, it should be consistent. Thanks @fainashalts